File: bbarena-model/src/main/java/net/sf/bbarena/model/event/game/StartDriveEvent.java
package net.sf.bbarena.model.event.game;

import net.sf.bbarena.model.Arena;
import net.sf.bbarena.model.util.Concat;

public class StartDriveEvent extends GameEvent {

	private static final long serialVersionUID = -1919966604189789283L;

	public StartDriveEvent() {
		super();
	}

	@Override
	public void doEvent(Arena arena) {
	}

	@Override
	protected void undoEvent(Arena arena) {
	}

	@Override
	public String getString() {
		return Concat.buildLog(getClass());
	}

}

File: src/main/kotlin/bbarena/model/event/game/StartDriveEvent.kt
package bbarena.model.event.game

import bbarena.model.Arena
import bbarena.model.util.Concat

class StartDriveEvent : GameEvent() {

    public override fun doEvent(arena: Arena) {}

    override fun undoEvent(arena: Arena) {}

    override fun getString(): String {
        return Concat.buildLog(javaClass)
    }

    companion object {

        private val serialVersionUID = -1919966604189789283L
    }

}
